CoEpi/ui/symptomReport/FollowUp/Cough/CoughHowViewController.swift

```diff
@@ -56,8 +56,28 @@ class CoughHowViewController: UIViewController {
         ButtonStyles.applyUnselected(to: worseButtonLabel)
         ButtonStyles.applyUnselected(to: sameButtonLabel)
         
+        betterButtonLabel.addTarget(self, action: #selector(onTouchDown(to:)), for:.touchDown)
```

Comment:
Needs refactoring / better solution, but again, let's let it like this because of the SwiftUI migration.

Reply:
I'll make an issue to track refactoring that.